### PR TITLE
SEO-174654-JavaScript-Image-With-anchor-text-two

### PR DESCRIPTION
**Pull request by Omanyadick**
Hi @Aishwarya-Ganesan, Kindly review my PR below.
Title | Description
-- | --
|Task Category | Site Audit work Links with no anchor text|
|Content Task Link/Mail Screenshot | NA|
|Hotfix | Hotfix- https://github.com/syncfusion-content/javascript-docs/pull/505|
|Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187693/|
|Work link | https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7BFFBBFCD9-0FB1-48D9-8F89-2C6BE58849E3%7D&file=Dickson.xlsx&action=default&mobileredirect=true|

Changes Made | Reason for change | Reason for not identifying earlier
-- | -- | --
 Links with no anchor text| Aesthetic and Professional Appearance to users. |  

Regards
Dickson J

### CODE REVIEW

File: js/PdfViewer/Localization.md [hunk 1]
```diff
@@ -908,5 +908,5 @@ The following screenshot shows the PDF viewer with tooltip in German language.
 
 <https://www.syncfusion.com/downloads/support/directtrac/general/ze/PdfViewer_Localization338521894>
 
-![Localization](Localization_images/Localization_img1.png)
+![Localization in JavaScript PDF Viewer.](localization_images/localization-in-javascript-pdf-viewer.png)
```
Comment (Yvone-Atieno) [inline js/PdfViewer/Localization.md:911]: Check image Alt

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: js/PdfViewer/Localization.md (hunk 1 above)
Comment (Omanyadick) [reply js/PdfViewer/Localization.md]: Alt text resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**